docs: test (#246)
pactflow/docs.pactflow.io: website/docs/docs/scim/okta.md

title: Integration guide - Okta
---

Okta supports using SCIM to manage users and groups in a configured application. Users can be created, updated, or disabled via the SCIM API (Okta does not support deleting users). For more details, refer to
[connecting SCIM API to Okta](https://developer.okta.com/docs/guides/scim-provisioning-integration-connect/main/) and the [Okta SCIM documentation](https://developer.okta.com/docs/concepts/scim/).  

## Setting up a PactFlow app in Okta

> Check out our article on [connecting SCIM API to Okta](https://developer.okta.com/docs/guides/scim-provisioning-integration-connect/main/)!

Reviewer: This is not "our" article, it's an Okta developer guide. Suggest:
```suggestion
Check out the official Okta guide on [connecting the SCIM API](https://developer.okta.com/docs/guides/scim-provisioning-integration-connect/main/)!
```
